code/lightna/magento-os-frontend/template/page/header/account/dropdown-customer.phtml

```diff
@@ -0,0 +1,15 @@
+<?php
+/** @var Lightna\Engine\Data\Block $block */
+$links = $block->{'.'};
+?>
+<div class="dropdown__wrap">
+    <ul class="dropdown__content dropdown__content--account">
+        <?php foreach ($links as $link): ?>
+            <li>
+                <a href="<?= $link->url ?>" title="<?= phrase($link->label) ?>">
```

Comment:
@vesan-83 should there be any escaping for href attr?

Reply:
sure, it's html output, $link->url()